### PR TITLE
feat: Mark proposals as completed or closed.

### LINKED ISSUE #2561: Mark proposals as completed or closed.
This issue is for the decred/politeia#1480 frontend implementation.

An admin needs the ability to set the proposal billing status as completed or closed.

### PR DESCRIPTION
**Pull request by amass01**
This commit allows admins to set a billing status of an approved
proposal by adding a new `Set Billing Status` button which is displayed
in the proposal card for admins for approved proposals.

---

Closes #2561.

### CODE REVIEW

**Comment by amass01** [comment]
Needs a commit message & e2e tests coverage.

**Comment by amass01** [comment]
Added e2e test coverage and commit message updated.

**Comment by lukebp** [comment]
This should be updated once decred/politeia#1515 is in.

An admin should only be able to set the billing status of proposals that have a proposal status of `active`.

File: teste2e/cypress/e2e/admin/proposals.js [hunk 1]
```diff
@@ -134,4 +134,55 @@ describe("Admin proposals actions", () => {
       }
     );
   });
+
+  it.only("Should allow admins to set the billing status of an approved propsoal", () => {
```
Comment (amass01) [inline teste2e/cypress/e2e/admin/proposals.js:138]: remove that only!

**Comment by amass01** [comment]
@lukebp Needs your final OK.

**Review by lukebp — CHANGES_REQUESTED** [review]
1. The button doesn't go away on the once you set the billing status.  It let's you attempt to set it again and returns an error.

image

2. The inconsistency between the button text `Set Billing Status` and the modal text `Set billing status` should be corrected.

There currently isn't any way to tell that the billing status of the proposal was updated.  I'm not going to ok this until we can test that functionality.  We need to be able to see a visual indication of the state change, which means #2609 needs to go in before this can go in.

**Comment by amass01** [comment]
Closing this PR as we need the set billing status changes with proposal statuses
in order to verify that the proposal status of an proposal updates properly when
setting the billing status.

**Pull request closed without merging**